cypress/page/payment-step.page.ts

```diff
@@ -5,7 +5,7 @@ class PaymentPage {
 
     constructor() {
       this.payByBankWire = "p.payment_module > a.bankwire";
-      this.confirmOrderBtn = "p.cart_navigation > button.button.btn.btn-default.button-medium";
+      this.confirmOrderBtn = "#cart_navigation > button[type=submit]";
```

Comment:
if you look at the source code, the button has inside a span, you can use the span instead of button type, since it's generic.

However, take a look at: https://docs.cypress.io/api/commands/contains#Content and see if you can use the name of the button with the contains function ;)